client/app/modules/series/root/MasterSeriesSF.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { getAllSeriesRequest } from '../../../state/series/actions/getSeries'
+import { searchSeriesRequest } from '../../../state/series/actions/searchSeries'
```

Comment:
use aliases for main folders

Reply:
Also you can create a barrel to export all your actions

Reply:
In the Shared HOC i cant use alias because the mobile build can`t resolve it, i have to search how configure alias in expo app